content/docs/configuration/issuers.md

```diff
@@ -35,7 +37,7 @@ The following list contains all known cert-manager issuer integrations.
 [//]: # (Configuration docs)
 
 [config:venafi-enhanced-issuer]: https://docs.venafi.cloud/vaas/k8s-components/t-vei-install/
-[config:acme-issuer]: ./acme.md
+[config:acme-issuer]: ./acme/README.md
```

Comment:
This broken link was picked up by the link checker.
Not sure why it hadn't highlighted it in earlier PRs.
 * https://github.com/cert-manager/website/actions/runs/7487481176/job/20379944460